Makefile

```diff
@@ -55,6 +55,7 @@ check: check-llvm
 test: check-llvm needs-cairo2 build-alexandria runtime-ci
 	cargo test --profile ci --all-features
 
+# remove the sed once circuits are implemented.
```

Comment:
Which sed? I think it was already removed.

Reply:
Yes, Done!